ocs_ci/deployment/helpers/lso_helpers.py

```diff
@@ -256,9 +256,12 @@ def create_optional_operators_catalogsource_non_ga(force=False):
     if config.DEPLOYMENT.get("optional_operators_image"):
         for _dict in optional_operators_data:
             if _dict.get("kind").lower() == "catalogsource":
-                _dict["spec"]["image"] = config.DEPLOYMENT.get(
-                    "optional_operators_image"
-                )
+                if not config.DEPLOYMENT.get("optional_operators_image"):
+                    _dict["spec"]["image"] = ocp_ga_version
```

Comment:
`ocp_ga_version` variable will probably contain just version string, not full image url

Reply:
yes it is value like 4.19. which is used in option-operator catalogsource creation for the image details.
like,
`image: quay.io/openshift-qe-optional-operators/aosqe-index:v4.19`